tools/check-shellcheck

```diff
@@ -0,0 +1 @@
+shellcheck -x $(file --mime-type tools/* | sed -n 's/^\(.*\):.*text\/x-shellscript.*$/\1/p')
```

Comment:
You've been adding the command directly withing CMake so this looks like a leftover.

Reply:
Actually I first tried to get the quoting right in CMake but quicky gave up after my nightmares from 15 years ago about CMake shell script quoting bubbled up again :D So now I am calling this script from cmake